logger.go

```diff
@@ -26,10 +26,9 @@ func NewLogger() *Logger {
 
 func (l *Logger) ServeHTTP(rw http.ResponseWriter, r *http.Request, next http.HandlerFunc) {
 	start := time.Now()
-	l.Printf("Started %s %s", r.Method, r.URL.Path)
 
 	next(rw, r)
 
 	res := rw.(ResponseWriter)
-	l.Printf("Completed %v %s in %v", res.Status(), http.StatusText(res.Status()), time.Since(start))
+	l.Printf("%v | %v | \t %v | %v | %v %v", start.Format(time.RFC3339), res.Status(), time.Since(start), r.Host, r.Method, r.URL.Path)
```

Comment:
I'm +0 to this change.  I like that the format is more compact and includes more information.  My personal preference would be to replace the hardcoded format string with a template string that's configurable at middleware construction time, allowing for usage like:

``` go
logger := negroni.NewLogger()
logger.Format = "{{.StartTime}} | {{.Status}} | \t {{.Duration}} | {{.Hostname}} | {{.Method}} {{.Path}}\n"
n := negroni.New()
n.Use(logger)
```

I think that if such a change were made, it would make sense to have the above format (the same as proposed in this patch) as the default.